cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -468,9 +467,9 @@ func TestSimulateInvokeContractTransactionSucceeds(t *testing.T) {
 	assert.Nil(t, obtainedAuth.RootInvocation.SubInvocations)
 
 	// check the events
-	assert.Len(t, response.Events, 1)
+	assert.Len(t, response.Events, 3)
```

Comment:
for consideration, maybe add comment or assert message, statying why 3 events are expected?